src/main/java/duke/userinterface/ConsoleInterface.java

```diff
@@ -0,0 +1,309 @@
+package duke.userinterface;
+
+import duke.task.*;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Provides functions to interface with user via standard input and standard output.
+ */
+public class ConsoleInterface {
+    public static final String TASKS_DIRECTORY_PATH = "./data/";
+    public static final String TASKS_FILENAME = "tasks.txt";
```

Comment:
A next step would be to use a config file to specify this rather than hardcoding the path and name.